README.md

```diff
@@ -0,0 +1,73 @@
+# Jaeger Spark dependencies
+
+This is a Spark job that will collect spans from your datastore, analyze links between services,
```

Comment:
Should this emphasize that only Cassandra is currently supported?

Also, I would make it clear that this will only work with data collected via Jaeger client libraries using the default model of single-host spans, i.e. it will not work with spans collected from most Zipkin or jaeger-configured-as-zipkin libraries that share the span ID between client and server.